Gems/LmbrCentral/Code/include/LmbrCentral/Shape/ShapeComponentBus.h

```diff
@@ -174,6 +191,19 @@ namespace LmbrCentral
             return false;
         }
 
+        /// Get the translation offset for the shape relative to its entity.
+        virtual AZ::Vector3 GetTranslationOffset()
```

Comment:
nit: Would be great to add `const` to this (I realize it's a Bus interface but still helps in this case I think and is worth adding - in an emergency people can always use `mutable` 😝)

Reply:
Done.